### ISSUE TITLE
Lightroom masking feature

### ISSUE THREAD

**Issue by aleks01100001** [ReVanced/revanced-patches#2745]
### Type

Error at runtime

### Bug description

When we patch the app with the "Disable mandatory login" patch, the app fails to download the masking features that are select subject and select sky.

### Steps to reproduce

1. Patch app with "Disable mandatory login" selected
2. Go to the masking feature while editing an image and try to download the automatoc modes

### Relevant log output

```shell
INFO: Reading dex files                                 │
│ INFO: Decoding AndroidManifest.xml only, because resour │
│ INFO: Skipping Change package name: Excluded by default │
│ INFO: Skipping Enable android debugging: Excluded by de │
│ INFO: Skipping Export all activities: Excluded by defau │
│ INFO: Skipping Predictive back gesture: Excluded by def │
│ INFO: Skipping Remove screen capture restriction: Exclu │
│ INFO: Skipping Remove screenshot restriction: Excluded  │
│ INFO: Skipping Spoof wifi connection: Excluded by defau │
│ INFO: Executing patches                                 │
│ INFO: Disable mandatory login succeeded                 │
│ INFO: Unlock premium succeeded                          │
│ INFO: Not compiling resources because resource patching │
│ INFO: Writing modified dex files                        │
│ INFO: Aligning base.apk to base-Revanced_aligned.apk    │
│ INFO: Signing base-Revanced_aligned.apk to base-Revance │
│ INFO: Found existing keystore: revancify.keystore       │
│ INFO: Copying base-Revanced_signed.apk to base-Revanced │
│ INFO: Cleaned up cache directory                        │
│ INFO: Finished
```

### Additional context

But if we do not select the mandatory login patch then the app downloads the files without a problem

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by reisxd** [comment]
That patch does not exist in official patches. You're using unofficial tools.

**Comment by aleks01100001** [comment]
> That patch does not exist in official patches. You're using unofficial tools.

yes, its one of the pre-released ones, v2.188.0-dev.4 to be exact. I just wanted to report that bug so this feature works correctly when it comes out in one of the official releases.

**Comment by oSumAtrIX** [comment]
Downloading requires logins.
